### PR TITLE
Upgrade: Upgrade doc for v2.1.0

### PR DESCRIPTION
**Pull request by Madhu-1**
This PR updates the upgrade document from v2.0.x to v2.1.0

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

File: docs/ceph-csi-upgrade.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 - [Ceph-csi Upgrade](#ceph-csi-upgrade)
   - [Pre-upgrade considerations](#pre-upgrade-considerations)
-  - [Upgrading from v1.2.x to v2.0.0](#upgrading-from-v12x-to-v200)
+  - [Upgrading from v1.2.x to v2.0.1](#upgrading-from-v12x-to-v201)
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md:5]: maybe drop the last component of the version? Use v2.0.x as minor 2.0 updates should not need any different steps

File: docs/ceph-csi-upgrade.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
...
+  - [Upgrading from v2.0.x to v2.1.0](#upgrading-from-v20x-to-v210)
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md:6]: prepare for v2.1.1 minor bugfix update too, just use v2.1.x

File: docs/ceph-csi-upgrade.md [hunk 3]
```diff
@@ -44,9 +45,15 @@ To avoid this issue in future upgrades, we recommend that you do not use the
 fuse client or rbd-nbd as of now.
 
 This guide will walk you through the steps to upgrade the software in a cluster
-from v1.2.x to v2.0.0
+from v2.0.x to v2.1.0
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md:48]: Probably just use "from v2.0 to v2.1" and leave the last component out

File: docs/ceph-csi-upgrade.md [hunk 4]
```diff
@@ -44,9 +45,15 @@ To avoid this issue in future upgrades, we recommend that you do not use the
...
 
-## Upgrading from v1.2.x to v2.0.0
+## Upgrading from v1.2.x to v2.0.1
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md:50]: Here too

File: docs/ceph-csi-upgrade.md [hunk 5]
```diff
@@ -44,9 +45,15 @@ To avoid this issue in future upgrades, we recommend that you do not use the
...
+
+Refer
+[upgrade-from-1.2.x-v2.0.1](https://github.com/ceph/ceph-csi/blob/v2.0.1/docs/ceph-csi-upgrade.md)
+to upgrade from cephcsi 1.2.x to v2.0.1
+
+## Upgrading from v2.0.x to v2.1.0
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md:56]: and here, and everywhere else

File: docs/ceph-csi-upgrade.md [hunk 6]
```diff
@@ -56,11 +63,11 @@ that will not be supported in the official releases. Builds from the master
 branch can have functionality changed and even removed at any time without
 compatibility support and without prior notice.
 
-git checkout release v2.0.0 branch
+git checkout release v2.1.0 branch
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md:66]: the branch should be release-2.1 and the tag will be v2.1.0

File: docs/ceph-csi-upgrade.md [hunk 7]
```diff
@@ -56,11 +63,11 @@ that will not be supported in the official releases. Builds from the master
...
 
 ```bash
 [$] git clone https://github.com/ceph/ceph-csi.git
-[$] git check v2.0.0
+[$] git check v2.1.0
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md:70]: check -> checkout - and you're checking out the tag here. not a branch

**Comment by Madhu-1** [comment]
@nixpanic done PTAL

File: docs/ceph-csi-upgrade.md [hunk 8]
```diff
@@ -56,11 +63,11 @@ that will not be supported in the official releases. Builds from the master
 branch can have functionality changed and even removed at any time without
 compatibility support and without prior notice.
 
-git checkout release v2.0.0 branch
+git checkout v2.1.0 tag
 
 ```bash
 [$] git clone https://github.com/ceph/ceph-csi.git
-[$] git check v2.0.0
+[$] git checkout v2.1.0
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md:70]: the order of `checkout` and `cd` is not correct

Comment (Madhu-1) [reply docs/ceph-csi-upgrade.md]: done

**Comment by Madhu-1** [comment]
@MergifyIO refresh

**Pull request merged**